fus.c

```diff
@@ -394,6 +401,11 @@ filelist_loadcb (Pool     *pool,
   fpath = repodata_lookup_str (data, SOLVID_META, REPOSITORY_REPOMD_LOCATION);
 
   fp = solv_xfopen (fpath, 0);
+  if (!fp)
+    {
+      g_info ("Could not open filelists %s: %s", fpath, g_strerror (errno));
+      return 0;
```

Comment:
All other places return `NULL`.

Reply:
I can return NULL here because this is a callback and libsolv expects either a 0 or 1 to know if the loading was successfull or not.

Reply:
Ah, that makes sense.